### ISSUE TITLE
Navigo fails precompile in Rails app

### ISSUE THREAD

**Issue by stoem** [krasimir/navigo#126]
We use a Rails app with Navigo. When precompiling the JS assets for the project the rake task fails with
`
app/assets/javascripts/navigo/index.js
ExecJS::ProgramError: Unexpected token: operator (>) (line: 42, col: 37, pos: 1135)
`
This is this line
` .reduce((params, value, index) => {...`
in 
`regExpResultToParams`

Now I'm a bit of a JS noob so I do not know how to rewrite the => so that this will pass the precompile.
Basically I need to remove the > sign.

Not sure if this is a bug but since all other JS files pass the precompile I figured it's worth looking at.

**Comment by stoem** [comment]
Damn, I just noticed this => syntax is used throughout the Navigo.js file.

I guess I need to learn this syntax and work out how to remove all the instances of => ?

**Comment by krasimir** [comment]
@stoem please use the files under the [lib](https://github.com/krasimir/navigo/tree/master/lib) folder. They are precompiled and do not use the arrow syntax.
